### DISCUSSION TITLE
How to access UW library off-campus with BIG-IP Edge Client?

### DISCUSSION THREAD

**Discussion by AHuffmyer** [RobertsLab/resources#1219] [Q&A]
Hi all! I am having trouble accessing UW libraries and get an error with "no off-campus access" when I try to access a paper. Has anyone else had this problem? Should I use BIG-IP Edge Client? Thanks in advance for any advice!

**Comment by kubu4** [comment]
I have no idea what that is. But, I'd use this as a viable alternative:

https://sci-hub.do/

I usually just use that as searching is faster and easier.

**Comment by sr320** [comment]
Edge client is part of Husky Onnet 

see https://www.lib.washington.edu/help/connect

**Reply by sr320** [reply]
Based on your error. I think this is the issue. If you want to drop into SH later we can look at .. I have noticed library content is increasingly harder to access for variety of reseasons
